Keas.Mvc/Services/IdentityService.cs

```diff
@@ -37,5 +40,37 @@ public IdentityService(IOptions<AuthSettings> authSettings)
                 return results.responseData.results.Count > 0 ? results.responseData.results[0].iamId : string.Empty;
             }
         }
+
+        public async Task<User> GetUser(string id, string email)
+        {
+            // return info for the user identified by this email address in IAM
+            using (var client = new HttpClient())
+            {
+                
+                client.BaseAddress = new Uri("https://iet-ws.ucdavis.edu/api/iam/people/");
```

Comment:
@jcubbage why manually grab it instead of using the IET webservice SDK?

Reply:
Cause I didn't that was a thing! Should we update GetUserID to use that as well? That code is from you 8 months ago.

Reply:
@jcubbage ha, that's funny i guess this project was the inspiration for me going out and writing the SDK.  Ok, let's leave this as is, we can go back and change it later.  BTW the SDK is https://github.com/ucdavis/ietws